crates/valence_protocol/src/packets/s2c.rs

```diff
@@ -608,6 +608,10 @@ pub mod play {
         pub blocks: &'a [VarLong],
     }
 
+    #[derive(Clone, Debug, Encode, EncodePacket, Decode, DecodePacket)]
+    #[packet_id = 0x40]
+    pub struct SelectAdvancementsTab<'a>(Option<Ident<&'a str>>);
```

Comment:
The inner value is not public. (As a side note, I'm probably going to change all the tuple structs into structs with named fields for consistency.)

Reply:
I can change all of them real quick if you would like.

Reply:
That'd be great.